conf/locale/locale_en-US.ini

```diff
@@ -729,6 +729,7 @@ settings.site = Official Site
 settings.update_settings = Update Settings
 settings.change_reponame_prompt = This change will affect how links relate to the repository.
 settings.advanced_settings = Advanced Settings
+settings.token_name_exists = Token with same name already exists.
```

Comment:
Hmm sorry I know it is somewhat confusing, the line should actually be added after/around here:

https://github.com/gogs/gogs/blob/99f7b6c1f37e8c6e78009c5793652e402df6b9a4/conf/locale/locale_en-US.ini#L370

FYI, `settings.` indicate the section name, then followed by the key name. The place where you added is actually `repo.settings.`, which is not what we want.